commitlint.config.ts

```diff
@@ -0,0 +1,37 @@
+import { RuleConfigSeverity } from "@commitlint/types";
+
+import type { UserConfig as CommitlintUserConfig } from "@commitlint/types";
+
+/**
+ * `commitlint` configuration.
+ * @see https://commitlint.js.org/#/reference-configuration
+ */
+const commitlintConfig: CommitlintUserConfig = {
+  extends: ["@commitlint/config-conventional"],
+  // whether to use default ignore rules
+  defaultIgnores: true,
+  // rule overrides
+  rules: {
+    // specify valid types
+    "type-enum": [
+      RuleConfigSeverity.Error,
+      "always",
+      [
+        "build", // build system, dependencies, tooling
+        "chore", // maintenance
+        "ci", // continuous integration
+        "docs", // update documentation
+        "feat", // add a feature (option 1 of 2)
```

Comment:
Also appreciate you adding `feat` even though you really didn't need to lol.

Reply:
Np! Closer in alignment to the [Conventional Commits spec](https://www.conventionalcommits.org/en/v1.0.0/) anyways!